pai-management/doc/cluster-bootup.md

```diff
@@ -76,7 +76,23 @@ sudo docker run -itd \
 sudo docker exec -it dev-box /bin/bash
 ```
 
-##### (3) Go to pai-management working dir
+##### (3) Check out a latest release branch of OpenPAI
+
+```bash
+cd /pai
+# please go to https://github.com/Microsoft/pai/releases to checkout a latest release.
+# checkout a release branch. For example: v0.x.y
+git checkout v0.x.y
```

Comment:
May need to run `git fetch --tags` first

Reply:
root@phltestaux1:/pai# git fetch  --tags v0.6.0
fatal: 'v0.6.0' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
root@phltestaux1:/pai# git fetch  --tags v0.6.1
fatal: 'v0.6.1' does not appear to be a git repository
fatal: Could not read from remote repository.

or:
root@phltestaux1:/pai# git fetch --tags v0.6.y
fatal: 'v0.6.y' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.